fixes #154: chore: deploy scripts should read private key from env var

appleseed-iii opened the pull request:
## Motivation

Fix per #154

varunsrin commented on script/BundleRegistry.s.sol:
```diff
@@ -15,13 +15,11 @@ contract BundleRegistryScript is Script {
 
-    // TODO: Always update this to the address of the private key used to deploy the contracts on the network
-    address constant DEPLOYER = address(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266); // anvil deployer
+    address DEPLOYER = vm.addr(vm.envUint("PRIVATE_KEY"));
```
if these aren't compile time constants anymore, we should lower case them

varunsrin reviewed (changes requested):
Looks great! 

I had one small comment and you'll need to sign your commits before Github will allow a merge in this repo: https://github.com/farcasterxyz/hubble/blob/main/CONTRIBUTING.md#22-signing-commits

appleseed-iii commented:
hey @varunsrin thanks. I force pushed my history to add the GPG signed commits. I also addressed your feedback in https://github.com/farcasterxyz/contracts/pull/193/commits/f36b4c2c5b3238c357549ad5625fb8003ff4b785

varunsrin commented:
thank you! whats your farcaster username?
